### DISCUSSION TITLE
Inbox support. DMs

### DISCUSSION THREAD

**Discussion by Clay-Ferguson** [undyingwraith/ipsm#12] [Ideas]
Supporting a PubSub topic format like this:

`/ipsm/$PKHash/inbox`

would allow direct messages between users. The $PKHash would be the hash of the PublicKey of the recipient. Message would be asymmetric encrypted using the PublicKey of the recipient to, so that only the recipient can read the message, if it's intended to be private.

Recipients can of course configure their system to only accept messages from their "Allowed Keys" list if they want, and of course there can be a list of keys each user maintains as "Blocked Keys" well, which always get ignored. But the Allows/Blocked keys is independent of this `inbox` concept, and would apply to all PubSub 'reads'

**Comment by undyingwraith** [comment]
I thought it would automatically link it. But guess not.
https://github.com/undyingwraith/ipsm/issues/15
